charts/opensearch/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.6.2
+version: 2.6.3
 
```

Comment:
Can you please update the `Unreleased` section as well?

Reply:
There is no Unreleased section in Chart.yaml. I updated the CHANGELOG.md